src/api/subscriptions.js

```diff
@@ -219,6 +226,15 @@ export let OffersRequest;
 export let LoginRequest;
 
 
+/**
+ * @typedef {{
+ *   token: (string|undefined),
```

Comment:
remove line 231-232 and replace with 'autoLoginHint: (string|undefined)' instead?

Reply:
done